### DISCUSSION TITLE
Is Python 3.12 officially supported?

### DISCUSSION THREAD

**Discussion by xyzzy-plugh-plover** [aws/aws-cli#8299] [Q&A] [answered]
Is Python 3.12 officially supported?

#8258 updated the CI to use 3.12, but [README.rst](https://github.com/aws/aws-cli/blob/develop/README.rst) doesn't list 3.12 as supported.

If 3.12 isn't yet supported, does anyone know when it will be?

If it is supported, then `README.rst` should be updated…

**Comment by tim-finnigan** [comment, marked as answer]
Thanks @xyzzy-plugh-plover for pointing this out - yes Python 3.12 is currently supported in v1 (and will be soon for v2)- the README was just recently updated: https://github.com/aws/aws-cli/pull/8303

**Reply by xyzzy-plugh-plover** [reply]
Thanks
